packages/fluentui/react-northstar/test/specs/components/Dropdown/DropdownSearchInput-test.tsx

```diff
@@ -3,9 +3,11 @@ import { isConformant } from 'test/specs/commonTests';
 
 describe('DropdownSearchInput', () => {
   isConformant(DropdownSearchInput, {
+    testPath: __filename,
     constructorName: 'DropdownSearchInput',
     hasAccessibilityProp: false,
     handlesAsProp: false,
+    disabledTests: ['as-renders-fc', 'as-passes-as-value', 'as-renders-html', 'as-renders-react-class'],
```

Comment:
This part can be reverted if you centralize `handlesAsProp` handling as suggested above